CHANGELOG.md

```diff
@@ -3,8 +3,156 @@ All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
 ## [Unreleased]
-- Leader responds to learner read index message. (https://github.com/pingcap/raft-rs/pull/220)
-- Snapshot may lose some applied results. (https://github.com/tikv/tikv/pull/4716)
+
+## [3.0.0]
+
++ Engine
+  - Introduce Titan, a key-value plugin that improves write performance for
```

Comment:
I think it is better to add some links to these things.

Reply:
The file is just checked out from release 3.0 branch. The content was copied from release notes. I think they were linked in the past release already.